Reformat and update docs (#66)
primer/presentations: content/design-guidelines/images.mdx

---
title: Image use
---

When using a screenshot, it’s best to match the background. We suggest to take a screenshot on a similar color background or use a PNG with a transparent background as this will look the most seamless.

> ## Usage guidelines:

Reviewer: ```suggestion
## Usage guidelines
```

<DoDontContainer>
  <Do>
    <img
      alt="image-white"
      src="https://user-images.githubusercontent.com/10384315/54647439-b32e1100-4a5f-11e9-8d85-a4a2e64319ca.png"
    />
    <Caption>Use images with a similar or transparent background</Caption>
  </Do>
  <Dont>
    <img
      alt="image-black"
      src="https://user-images.githubusercontent.com/10384315/54647548-1f107980-4a60-11e9-9640-8f1721609e44.png"
    />
    <Caption>Don't use images with opposite backgrounds to the slide background</Caption>
  </Dont>
</DoDontContainer>

### 📍Tip: Seamless image backgrounds

> If you find yourself needing to use a specific image but the slide background doesn't match, you can change the background color to match the image's background color.

Reviewer: ```suggestion
If you need to use a specific image but the slide background doesn't match, you can change the background color to match the image's background color.
```

> ![bg-change](https://user-images.githubusercontent.com/10384315/56434794-05469a00-628b-11e9-8447-93bb44facbec.gif)

Reviewer: Can you add alt text to this gif? Or a caption?

## Adding images

> Images can be easily added to any presentation by simply drag and dropping, copy and pasting, or inserting from the toolbar in Keynote, Google Slides, and Figma.

Reviewer: Let's try to never say: easy, simple, just 😄 

I find "drag and dropping" weird, but leaving it up to you, and my 1 second research [result](https://english.stackexchange.com/questions/92755/drag-dropping-or-dragging-dropping).

```suggestion
Images can be added to any presentation by drag and dropping, copy and pasting, or inserting from the toolbar in Keynote, Google Slides, and Figma.
```

|            Click the `Replace`             | ![Screenshot 2019-04-02 13 00 23](https://user-images.githubusercontent.com/10384315/55432266-4d558500-5547-11e9-88b2-21818e9182c3.png) |
|                                            |
|   Select the new image and click `Open`    |       ![keynote-replace-2](https://user-images.githubusercontent.com/10384315/55841327-9ecfb800-5ae3-11e9-8277-2c0aa79a0f3a.gif)        |

### Adding images in Google Slides

|    Source     |                                       Description                                        |                                                           Example                                                           |
| :-----------: | :--------------------------------------------------------------------------------------: | :-------------------------------------------------------------------------------------------------------------------------: |
|    Browser    | Drag and drop any image from any tab in your web browser into the tab with Google Slides | ![google-add-image-1](https://user-images.githubusercontent.com/10384315/55434040-7d9f2280-554b-11e9-926c-690b8a001c3e.gif) |

> | Local Machine |              In the toolbar select `Insert > Image > Upload from computer`               | ![google-add-image-2](https://user-images.githubusercontent.com/10384315/55434041-7d9f2280-554b-11e9-844f-8d019a7d8466.gif) |

Reviewer: ```suggestion
| Local machine |              In the toolbar select `Insert > Image > Upload from computer`               | ![google-add-image-2](https://user-images.githubusercontent.com/10384315/55434041-7d9f2280-554b-11e9-844f-8d019a7d8466.gif) |
```

|    By URL     |        In the toolbar select `Insert > Image > By URL` and paste the image's URL         | ![google-add-image-3](https://user-images.githubusercontent.com/10384315/55434043-7d9f2280-554b-11e9-9c2b-520055cbbfb5.gif) |

### Replacing images in Google Slides

|                               Step                                |                                                                         Example                                                                          |
| :---------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------: |

> | With the image selected, click the `Replace Image` in the toolbar |         ![Screenshot 2019-04-02 13 35 49](https://user-images.githubusercontent.com/10384315/55434339-3feec980-554c-11e9-87aa-607f84827120.png)          |

Reviewer: ```suggestion
| With the image selected, click `Replace Image` in the toolbar |         ![Screenshot 2019-04-02 13 35 49](https://user-images.githubusercontent.com/10384315/55434339-3feec980-554c-11e9-87aa-607f84827120.png)          |
```
